W-16778879 HL7 Connector 4.3.0 Release (#2698)
mulesoft/docs-connectors: hl7/4.3/modules/ROOT/pages/hl7-connector-reference.adoc

In either case the error is reported in an ERR segment. |  false |
| Fail when segment out of order in message set a| Boolean a| Whether to reject a message when a segment in a message set is out of order:

* If `true`, a message with this error is rejected.
* If `false` and the segment can be reordered, the message is accepted.

In either case the error is reported in an ERR segment. |  false |
| Fail when unused segment included in message set a| Boolean a| Whether to reject a message when a segment marked as unused is included in a message set: 

* If `true`, a message with this error is rejected.
* If `false`, the message is accepted and the unused segment is ignored.

In either case the error is reported in an ERR segment. |  false |
| Fail when too many repeats of segment a| Boolean a| Whether to reject a message when a segment occurs too many times in a message set:

* If `true`, a message with this error is rejected.
* If `false`, the message is accepted.

In either case, the error is reported in an ERR segment. |  false |
| HL7 Message segment terminator a| Enumeration, one of:

** CR
** LF |  Value for the segment terminator. |  CR |

> | Enforce Acknowledgment ERR Segment repeatability a| Boolean a| Whether to enforce the ERR Segment repeatability (count) in the Acknowledgment:

Reviewer: ```suggestion
| Enforce Acknowledgment ERR Segment repeatability a| Boolean a| Whether to enforce the ERR Segment repeatability (count) in the acknowledgment:
```

> * If `true` and the ERR segment count is not 1, errors will be added to the ERR-01 (ELD) in a single ERR segment.

Reviewer: ```suggestion
* If `true` and the ERR segment count is not 1, errors are added to the ERR-01 (ELD) in a single ERR segment.
```

> * If `false`, every error will be added to the ERR list.

Reviewer: ```suggestion
* If `false`, every error is added to the ERR list.
```
